app/javascript/src/components/Dashboard/Notes/Utils.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
```

Comment:
Wrong naming followed for the file. It's a lack of self-review that is evident. Otherwise this would've been caught. Thus attaching the self-review document once again and watch the video once again if need be:
- https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Note: This is a warning from my end. Further lack of self-review will have consequences.

Also read: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react

Reply:
Sorry for the mistake, I did check these statements for quite some time. I shall find out what is wrong fix it ASAP.

Reply:
@eshaanks The filename is what is wrong. You have named this file as if it's a React component, whereas it's not.

Reply:
Will rectify this.

Reply:
I am encountering some sort of bug where I have the file name in lower case (utils.js) in my local file directory but when I am pushing it still is showing the file name as 'Utils.js' . What may be the cause for this?

Reply:
It was fixed by changing my git config via this command `git config core.ignorecase false`. It seems the changes in file name are reflected properly.

Reply:
It seems that I needed to add some new lines to my git config so file name changes like changing the case will be caught.`git config core.ignorecase false`, is this okay?

Reply:
@eshaanks That settings seems unnecessary. You should be moving the file. Refer: https://stackoverflow.com/a/20907647

Reply:
Make sure to revert the git config change that you have made:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/31244999/190591739-db5da919-db74-4ac5-8338-7eb4ef62aada.png">

Reply:
I have removed the faulty settings that caused a duplicate Utils.jsx to be spawned within the repo. I have deleted the duplicate and renamed the original to 'utils.jsx' .